js/apps/events-page/dataServices/eventsService.js

```diff
@@ -49,6 +49,7 @@
 				$http.get(CONSTANTS.baseUrl + CONSTANTS.serviceUrls.events + '/' + id)
 					.then((response) => {
 						if (response.data) {
+							response.data.Description = response.data.Description.replace(/<[\w/]+>/g, '');
```

Comment:
Do we need to check if this exists?

Reply:
It might not be completely necessary, but it sure makes me feel better.